Cargo.lock

```diff
@@ -415,29 +459,29 @@ dependencies = [
 
 [[package]]
 name = "pyo3-build-config"
-version = "0.23.4"
+version = "0.23.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1cd3927b5a78757a0d71aa9dff669f903b1eb64b54142a9bd9f757f8fde65fd7"
+checksum = "94f6cbe86ef3bf18998d9df6e0f3fc1050a8c5efa409bf712e661a4366e010fb"
 dependencies = [
  "once_cell",
  "target-lexicon",
 ]
 
 [[package]]
 name = "pyo3-ffi"
-version = "0.23.4"
```

Comment:
It looks like you ran an update while adding the dependencies. Revert this file and  just run something along the lines of `cargo add crate --features feature_foo` instead.

Let's try to keep this PR as small as possible in terms of dependencies added.